megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json

```diff
@@ -11232,7 +11232,7 @@
             "$id": "#/properties/POWERSHELL_POWERSHELL_CLI_EXECUTABLE",
             "default": [
                 [
-                    "powershell"
+                    "pwsh"
```

Comment:
When did the default value change? This PR applies this change to the docs.

Reply:
this is when you run `bash build.sh` on windows :)
it reverts back the next time it's run on linux ^^